### ISSUE TITLE
Generate enums as list

### ISSUE THREAD

**Issue by sporto** [dillonkearns/elm-graphql#77]
We often use enums as dropdown options e.g.

```
type Severity
    = Low
    | Medium
    | High
```

Needs to a list of `[Lost, Medium, High]` so we can create a dropdown for the user. Elm doesn't allow iterating over union types. 

But maybe this lib could fix this. What about if the package also generated enums as a list: 

```
list = [Lost, Medium, High]
```

In this way if we add a new enum variant in the BE, it will show in the FE dropdown without us having to add it to a list manually.

**Comment by dillonkearns** [comment]
Great idea, thanks for the suggestion! Should be a super easy change, and there's a good payoff. I'll work on this when I get some time 👍

**Comment by dillonkearns** [comment]
Done! This is now released in NPM version 1.0.6. Thanks for the great suggestion!

Here's an example: https://github.com/dillonkearns/elm-graphql/blob/master/examples/src/Swapi/Enum/Episode.elm#L23-L25.
